From f75d234990969c6a5288792502da0a25f51a6d63 Mon Sep 17 00:00:00 2001
From: Jose Luis Blanco-Claraco <joseluisblancoc@gmail.com>
Date: Sat, 21 Jun 2025 00:20:46 +0200
Subject: [PATCH] Debian rules.em: disable Boost features

---
 debian/rules | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/debian/rules b/debian/rules
index 8e7edce..d9e3ae1 100755
--- a/debian/rules
+++ b/debian/rules
@@ -38,6 +38,8 @@ override_dh_auto_configure:
 		-DCMAKE_INSTALL_PREFIX="/opt/ros/rolling" \
 		-DCMAKE_PREFIX_PATH="/opt/ros/rolling" \
 		-DGTSAM_USE_SYSTEM_EIGEN=ON \
+		-DGTSAM_ENABLE_BOOST_SERIALIZATION=OFF \
+		-DGTSAM_USE_BOOST_FEATURES=OFF \
 		$(BUILD_TESTING_ARG)
 
 override_dh_auto_build:
-- 
2.34.1

